src/lib.rs

```diff
@@ -56,10 +56,12 @@
 #![warn(rust_2018_idioms)]
 #![warn(rustdoc::all)]
 
+/*
 #[cfg(any(test, miri))]
 pub(crate) const R: usize = 4;
 #[cfg(not(any(test, miri)))]
 const R: usize = 8;
+*/
```

Comment:
Just remove this rather than comment it out :)

Reply:
Yeah that was mostly for testing and keeping the current default settings